src/gsi/stppm2_5.f90

```diff
@@ -255,6 +256,97 @@ subroutine stppm2_5(pm2_5head,rval,sval,out,sges,nstep)
 
     end if
 
+    if ( laeroana_fv3smoke) then
+       pm2_5ptr => pm2_5Node_typecast(pm2_5head)
+
+       do while (associated(pm2_5ptr))
+          if(pm2_5ptr%luse)then
```

Comment:
These if statements are confusing, where are the else/end ifs? Be consistent with indentation and perhaps add comments showing where the else/endif is.

Reply:
Yes. I will do it.